docs/rst/tutorials/core/deployment/large_data/large_data.rst

```diff
@@ -0,0 +1,245 @@
+
+Large Data Mode can be easily enabled using the ``FASTDDS_BUILTIN_TRANSPORTS`` environment variable, XML profiles, or code:
+
+.. code:: bash
+
+    export FASTDDS_BUILTIN_TRANSPORTS=LARGE_DATA
+
+This simplifies the configuration process and helps optimize large data transmission without needing extensive network adjustments.
+
+Large Data with Configuration options
+"""""""""""""""""""""""""""""""""""""
+
+In addition to the default setup, Large Data Mode offers advanced configuration options to enhance performance when handling large data, such as video streaming.
+Fast DDS provides these key parameters to adjust transport behavior in ``LARGE_DATA`` mode:
+
+- ``max_msg_size``: Maximum message size before fragmentation, up to (2^32)-1 B bytes for TCP and SHM, and 65500 KB for UDP.
+- ``sockets_size``: Size of the send and receive buffers; should match or exceed max_msg_size and defines the SHM size.
+- ``non_blocking``: Avoids application blocking when buffers are full, though some message loss may occur. Default is false.
+- ``tcp_negotiation_timeout``: Sets a timeout for TCP port negotiation, helping prevent initial message loss.
+
+Tuning these parameters for large messages and enabling non-blocking mode can optimize performance for high-bandwidth data flows like video.
+
+The following snippets show how to configure the ``LARGE_DATA`` mode:
+
+.. code-block:: bash
+
+    export FASTDDS_BUILTIN_TRANSPORTS=LARGE_DATA?max_msg_size=1MB&sockets_size=1MB&non_blocking=true&tcp_negotiation_timeout=50
+
+.. _tutorials_large_data_video_streaming_prerequisites:
+
+Prerequisites
+-------------
+
+It is required to have previously installed Vulcanexus using one of the following installation methods:
+
+* `Linux binary installation <https://docs.vulcanexus.org/en/latest/rst/installation/linux_binary_installation.html>`__
+* `Linux installation from sources <https://docs.vulcanexus.org/en/latest/rst/installation/linux_source_installation.html>`__
+* `Docker installation <https://docs.vulcanexus.org/en/latest/rst/installation/docker.html>`__
+
+Additionally, you need to have OpenCV installed.
+You can follow the `OpenCV documentation <https://docs.opencv.org/2.4/doc/tutorials/introduction/table_of_content_introduction/table_of_content_introduction.html#table-of-content-introduction>`__ for detailed installation instructions.
+
+You will also need the ROS package ``image_tools``, which can be installed by running:
+
+.. code:: bash
+
+    sudo apt-get install ros-humble-image-tools
+
+Lastly, ensure you have a working webcam connected to one of the hosts.
+
+Run this tutorial
+------------------
+
+UDP Streaming
+^^^^^^^^^^^^^
+
+In the first part of the tutorial, we will use the default UDP settings.
+Follow these steps:
+
+1. Source your Vulcanexus setup file on both computers:
+
+.. code:: bash
+
+    . <path to Vulcanexus install space>/setup.bash
```

Comment:
```suggestion
    source /opt/vulcanexus/jazzy/setup.bash
```

Reply:
I'd also move this first point to the prerequisites section just stating that you need to source your vulcanexus installation environment before running any ros 2 commands.

Reply:
I've added a note in the Prerequisites

Reply:
Add also the command to the note.